<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="01HW198525" name="com.TestSuite.Customer180.MyDesk_C1_021" tests="3" failures="3" timestamp="9 Mar 2017 12:50:47 GMT" time="462.275" errors="0">
  <testcase name="MyDesk_C1_021_Method" time="150.435" classname="com.TestSuite.Customer180.MyDesk_C1_021">
    <failure type="java.lang.AssertionError" message="null script failed, Exit_Interation_Comments - null.jpg for more details.">
      <![CDATA[java.lang.AssertionError: null script failed, Exit_Interation_Comments - null.jpg for more details.
	at org.testng.Assert.fail(Assert.java:94)
	at com.Engine.Reporter.exception(Reporter.java:107)
	at com.WebActions.WebActions.VerifyElementPresentAndClearType(WebActions.java:405)
	at com.BusinessModules.MyDesk.Common_Functions.enter_Exit_interaction_Comments(Common_Functions.java:373)
	at com.TestSuite.Customer180.MyDesk_C1_021.MyDesk_C1_021_Method(MyDesk_C1_021.java:150)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:963)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1142)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- MyDesk_C1_021_Method -->
  <testcase name="MyDesk_C1_021_Method" time="169.839" classname="com.TestSuite.Customer180.MyDesk_C1_021">
    <failure type="java.lang.AssertionError" message="null script failed, Exit_Interation_Comments - null.jpg for more details.">
      <![CDATA[java.lang.AssertionError: null script failed, Exit_Interation_Comments - null.jpg for more details.
	at org.testng.Assert.fail(Assert.java:94)
	at com.Engine.Reporter.exception(Reporter.java:107)
	at com.WebActions.WebActions.VerifyElementPresentAndClearType(WebActions.java:405)
	at com.BusinessModules.MyDesk.Common_Functions.enter_Exit_interaction_Comments(Common_Functions.java:373)
	at com.TestSuite.Customer180.MyDesk_C1_021.MyDesk_C1_021_Method(MyDesk_C1_021.java:150)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:963)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1142)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- MyDesk_C1_021_Method -->
  <testcase name="MyDesk_C1_021_Method" time="142.001" classname="com.TestSuite.Customer180.MyDesk_C1_021">
    <failure type="java.lang.AssertionError" message="null script failed, Exit_Interation_Comments - null.jpg for more details.">
      <![CDATA[java.lang.AssertionError: null script failed, Exit_Interation_Comments - null.jpg for more details.
	at org.testng.Assert.fail(Assert.java:94)
	at com.Engine.Reporter.exception(Reporter.java:107)
	at com.WebActions.WebActions.VerifyElementPresentAndClearType(WebActions.java:405)
	at com.BusinessModules.MyDesk.Common_Functions.enter_Exit_interaction_Comments(Common_Functions.java:373)
	at com.TestSuite.Customer180.MyDesk_C1_021.MyDesk_C1_021_Method(MyDesk_C1_021.java:150)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- MyDesk_C1_021_Method -->
</testsuite> <!-- com.TestSuite.Customer180.MyDesk_C1_021 -->
